README.md

```diff
@@ -34,6 +49,67 @@ The following verification utility will re-verify these numbers.
+
+To get started, populate the `.env` file with your information, such as your Google Voice number and Twitter tokens. For example:
+
+```bash
+BOT_NUMBER=<Google Voice number>
+ADMIN_CONTACT=<Your number>
+TWITTER_API_KEY=<Generated Twitter API Key>
+TWITTER_API_SECRET=<Generated Twitter API Secret>
+TWITTER_ACCESS_TOKEN=<Generated Twitter Access Token>
+TWITTER_TOKEN_SECRET=<Generated Twitter Access Token Secret>
+...
+
+```
+
+Note that to you do not need to fill out all of the environment variables yet. Many of these variables will have defaults. To use the defaults, simply remove the variable line. For example, if you would like to use the default timezone, you would remove the entire line:
+
+`DEFAULT_TZ=TZ database formatted name for a timezone. Defaults to US/Pacific`
+
+Once you have your phone numbers in the `.env` file, run `just register`.
+
+**WINDOWS USERS:** If you are on Windows, you can pass this into your `justfile` to allow it to use Powershell instead of using `sh`:
+
+```
+# use PowerShell instead of sh:
+set shell := ["powershell.exe", "-c"]
+```
+
+When running `just register`, it should prompt you to receive a CAPTCHA token:
+
+```bash
+> just register
+docker-compose --file=docker-compose.yml --file=docker-compose.override.yml run --rm cli ./register-number.sh
+Creating signal-scanner-bot_cli_run ... done
+Captcha required, please visit https://signalcaptchas.org/registration/generate.html to receive a captcha token
+```
+
+**Note:** If you do not get this response, double check your environment variables in the `.env` file.
+
+Visit the link provided in the response, which will prompt you to open Signal. You should be greeted with a blank webpage. You can close out of the prompt, as you do not require the client to be running to verify the CAPTCHA.
+
+On the blank webpage, run the inspect tool for your browser. In the Console logs, you should see a message like such:
+
+`>Launched external handler for 'signalcaptcha://<string> generate.html:1<string>'.`
+
+Copy the entire string that comes after `signalcaptcha://`. This will be the code you use to verify the CAPTCHA in your terminal. It should be a 484 character-long string.
+
+For more information on this step, you can check out this page here: [Registration with captcha](https://github.com/AsamK/signal-cli/wiki/Registration-with-captcha)
+
+Once successful, you will be asked to enter the verification value sent to your primary phone. This will be sent to the Google Voice number that you set as a value for the environment variable `BOT_NUMBER`.
+
+If the verification code is accepted, you will receive a return message stating that verification was successful, and that the Google Voice number can now be used:
+
+`>Verification succeeded, +<BOT_NUMBER> can now be used`
+
+Next, run `just up` to run your container. 
+
+To verify that everything is working, you can run `docker ps` to grab the container ID, and then run `docker logs <container-id>` to see the logs.
```

Comment:
```suggestion
To verify that everything is working, you can run `docker logs $(docker ps | grep "sdsa/signal-scanner-bot" | awk -F ' ' '{print $1}')` to see the logs.
```

Reply:
I think we would actually do best with a `just logs` recipe. It could look like this in the justfile:
```
# Print the logs from the service
logs *args:
	{{ DC }} logs -f {{ args }}
```

If we don't want to go that route, `docker-compose logs` will give the same result without having to parse `docker ps`.

Reply:
That's nice. I'm not too familiar with `just` recipes (only found out about them while working on this repo) so this is great to know! Similarly, happy to open a separate PR with this addition.